### ISSUE TITLE
Keyboard shortcuts dialog is missing a11y content in live version

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#80]
I just noticed that there is no accessible content in the keyboard shortcuts dialog. Not sure if this was intentional or accidental as the sim is not yet fully screen reader accessible.

However, we do have content for the dialog, so I would suggest we add it in.

See design document for A11y Content. There is no need to update to the latest version, just add in existing list items for the existing visual content.

A11y Design doc for Friction:
https://docs.google.com/document/d/1v7GM_dNZs86jcRn2X03D2t7HFDb4dQ1h4ayDGXTu63U/edit#heading=h.o6iy1plf4s2f

**Comment by terracoda** [comment]
@jessegreenberg could you assign to a developer? Not sure if a priority is needed from @ariel-phet? Assigning to @ariel-phet as well.

**Comment by ariel-phet** [comment]
@terracoda I am confused...if the sim is not yet fully screen reader accessible, why would we add this content into the dialog of the published version?

**Comment by jessegreenberg** [comment]
I don't think this was an oversight, but I could be wrong. The latest deploy of this sim included keyboard navigation and labels for the focusable items, like "Chemistry Book".

I believe this sim will get full description soon and I think that is the best time to include this content. Does that sound good @terracoda?

**Comment by terracoda** [comment]
@jessegreenberg, @ariel-phet I don't think it was an developer oversight, but I do think it was a design oversight on my part.

Static descriptions are meant to provide some access to sim interactions for people with screen readers - not full access, but some access. 

However, when a user opens the dialog, they only hear heading content and no description of the actual interactions.

As far as I understand the strings for the dialog are pretty straight forward to implement, so ideally for sims with keyboard dialogs, we should just provide the a11y content as well.

**Comment by terracoda** [comment]
@jessegreenberg, we could certainly wait for the implementation of full description. My concern was that with just the heading content, it seems a little broken. We can discuss at A11y Meeting.

**Comment by terracoda** [comment]
Closing this issue. Will implement with full description which is coming soon.
